Source/IntegrationTests/TestFiles/LitTests/LitTest/invariants/Performance.dfy

```diff
@@ -0,0 +1,27 @@
+// RUN: %verify --type-system-refresh --check-invariants --performance-stats=1 "%s" > "%t"
```

Comment:
Could you add a performance test like the one I described in this comment? https://github.com/dafny-lang/dafny/pull/6313#pullrequestreview-3132184002